src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/Editor_OutOfProc.cs

```diff
@@ -362,11 +365,17 @@ private TextSpan[] Deserialize(string[] v)
             }).ToArray();
         }
 
-        public void GoToDefinition()
-            => _editorInProc.GoToDefinition();
+        public void GoToDefinition(string viewName)
```

Comment:
```suggestion
        public void GoToDefinition(string expectedDefinitionViewName)
```

Since I admit otherwise the various uses weren't obvious what this was. Also should it be nullable which instead translates to "I expect the active view not to change because we're popping up the tool window?" Because otherwise the wait isn't doing anything in that case, and not sure how that's being handled in the LSP case?

Reply:
(random question: if we switch this to wait for a _window_ with your new helper...does that work?)

Reply:
Seems to work actually, though I had to change a few names (for example include [from metadata] in some).